src/api.ts

```diff
@@ -0,0 +1,387 @@
+import * as _ from "lodash";
+import * as querystring from "querystring";
+import * as requestModule from "request";
+import * as url from "url";
+import * as FormData from "form-data";
+
+import { Constants } from "./emulator/constants";
+import { FirebaseError } from "./error";
+import * as logger from "./logger";
+import * as responseToError from "./responseToError";
+import * as scopes from "./scopes";
+import * as utils from "./utils";
+
+export enum Method {
```

Comment:
I'd call it HttpMethod if we're going to have this at all

Reply:
Done